extension/js/content_scripts/webmail/setup-webmail-content-script.ts

```diff
@@ -285,6 +286,11 @@ export const contentScriptSetupIfVacant = async (webmailSpecific: WebmailSpecifi
       const acctEmail = await waitForAcctEmail();
       const { tabId, notifications, factory, inject } = await initInternalVars(acctEmail);
       await showNotificationsAndWaitTilAcctSetUp(acctEmail, notifications);
+      const authInfo = await AcctStore.authInfo(acctEmail);
+      if (authInfo.uuid) { // have auth email set
+        // todo: catch? ignore?
+        await (new AccountServer(acctEmail)).accountGetAndUpdateLocalStore(authInfo);
```

Comment:
@tomholub Do we need some special error handling when
1) uuid is not set
2) accountGetAndUpdateLocalStore throws an exception?
Should we toast a message and continue?

Reply:
ping @tomholub

Reply:
There are two types of endpoints to get client configuration from:
 - flowcrypt.com/api - authenticated, which uses uuid
 - fes.customer.com - unauthenticated

If you look inside of `AccountServer.accountGetAndUpdateLocalStore`, you'll notice that it's only using uuid for one of the cases.

I would remove the `if` in your code and instead try/catch:
  - catching `BackendAuthErr` and ignoring it, with a comment that user will see a prompt to log in during some other actions that involve backend, and therefore the value will eventually be filled, at which point it will work successfully (no need to wait and retry, it will just get redone next time user loads the page).
  - catching ApiErr.isNetErr and ignoring it
  - remaining errors should be toasted: `Failed to update FlowCrypt Client Configuration: e.message (e.type)`

Reply:
not sure what you mean by `e.type` -- we don't have this in this project.
Other changes have been applied.

Reply:
Oh, I meant the error instance. In case it says something meaningful like `NetworkError` or whatnot.

Reply:
Looks good to me 👍